contracts/SwapLibrary.sol

```diff
@@ -72,7 +81,17 @@ library SwapLibrary {
    * @param tokenIn The address of the token to be used as input for the swap.
    * @param tokenOut The address of the token to be received as a result of the swap.
    * @param amount The desired amount of output tokens (`tokenOut`) to be obtained from the swap.
-   * @param price The price at which the swap will occur.
+   * @param price Approximate amount of units of tokenIn required to acquire a unit of tokenOut.
+   *              It will be validated against the swap rate considering the maxSlippage.
+   *
+   * - Should have sufficient `tokenIn` to fulfill the desired output amount.
```

Comment:
Este texto no tiene que ir con un `@notice` o similar? Creo que si no es parte de `@param price`.

También estaría bueno separar las pre-condiciones y las post-condiciones. `The actual amount of input tokens...` es una post-condición o parte del `@returns`